src/metabase/cmd/copy.clj

```diff
@@ -382,15 +383,15 @@
     (classloader/require ns-symb))
   ;; make sure the source database is up-do-date
   (step (trs "Set up {0} source database and run migrations..." (name source-db-type))
-    (mdb.setup/setup-db! source-db-type source-data-source true))
+    (mdb.setup/setup-db! source-db-type source-data-source true false))
```

Comment:
Ha, I was thinking you might catch that. The only reason I didn't do it because I was lazy, but since you like keyword arguments (as I do) I will follow through with it

Reply:
I should also have said - fine for expedience, if you'll fix in a later PR. I suspected as much.

Reply:
yep, will do in a follow up PR